lib/core/core.module.ts

```diff
@@ -39,7 +39,7 @@ import { ToolbarModule } from './toolbar/toolbar.module';
 import { UserInfoModule } from './userinfo/userinfo.module';
 import { ViewerModule } from './viewer/viewer.module';
 import { FormModule } from './form/form.module';
-import { SidenavLayoutModule } from './sidenav-layout/sidenav-layout.module';
+import { LayoutModule as SidenavLayoutModule } from './sidenav-layout/layout.module';
```

Comment:
Can you please rename the module's folder also from sidenav-layout to layout (from this point on, this folder is supposed to containe every layout related general elements).